styles.css

```diff
@@ -1 +1,66 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+.container {
+    background-color: hsl(6,72%,89%);
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 150px 150px auto 150px 90px;
+    grid-template-areas:
+                      " header "
+                      " aside-left "
+                      " main "
+                      " aside-right "
+                      " footer ";
+}
+.item {
+    background-color: hsl(289,13%,42%);
+    border: 2px solid hsl(289,13%,32%);
+    color: white;
+    display:flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#item-one {
+    grid-area: header;
+
+}
+
+#item-two {
+    grid-area: aside-left;
+
+
+}
+
+#item-three {
+    grid-area: main;
+
+
+}
+
+#item-four{
+    grid-area: aside-right;
+
+
+}
+
+#item-five{
+    grid-area: footer;
+
+} 
+
+@media screen and (min-width:576px) {
+    .container {
+        background-color: hsl(6,72%,89%);
+        display: grid;
+        row-gap: 16px;
+        column-gap: 16px;
+        grid-template-columns: 1fr 2fr 1fr;
```

Comment:
Moet zijn: `grid-template-columns: repeat(3, 1fr);` of `grid-template-columns: 1fr 1fr 1fr;`